Pully Penelope/Assets/Scripts/PlayerMovement.cs

```diff
@@ -22,11 +30,37 @@ private void Update()
         positionChange = Vector3.zero;
         positionChange.x = Input.GetAxisRaw("Horizontal");
         positionChange.y = Input.GetAxisRaw("Vertical");
-
-        /*if (positionChange != Vector3.zero)
+        grabFloat = Input.GetAxisRaw("Jump");
+        if (handleName != null && grabFloat != 0)
         {
-            MovePlayer();
-        }*/
+            if (handleName == "DHandle")
```

Comment:
You are using a bunch of magic strings here, and it's hard to tell what they mean. Is there another script you can pull these values from?